modules/ROOT/pages/schema/constraints/list-constraints.adoc

```diff
@@ -0,0 +1,316 @@
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+======
+
+
+[[list-constraints-result-columns]]
+== Result columns for listing constraints
+
+.Listing constraints output
+[options="header", width="100%", cols="4m,6a,2m"]
+|===
+| Column | Description | Type
+
+| id
+| The id of the constraint. label:default-output[]
+| INTEGER
+
+| name
+| Name of the constraint (explicitly set by the user or automatically assigned). label:default-output[]
+| STRING
+
+| type
+| The ConstraintType of this constraint (`NODE_PROPERTY_UNIQUENESS`, `RELATIONSHIP_PROPERTY_UNIQUENESS`, `NODE_PROPERTY_EXISTENCE`, `RELATIONSHIP_PROPERTY_EXISTENCE`, `NODE_PROPERTY_TYPE`, `RELATIONSHIP_PROPERTY_TYPE`, `NODE_LABEL_EXISTENCE`, `RELATIONSHIP_SOURCE_LABEL`, `RELATIONSHIP_TARGET_LABEL`, `NODE_KEY`, or `RELATIONSHIP_KEY`). label:default-output[]
+
+| STRING
+
+| entityType
+| Type of entities this constraint represents (`NODE` or `RELATIONSHIP`). label:default-output[]
+| STRING
+
+| labelsOrTypes
+| The labels or relationship types of this constraint.
+The list returned will only include a single value (the name of the constrained node label or relationship type). label:default-output[]
+| LIST<STRING>
+
+| properties
+| The properties of this constraint, or `null` for node label existence, relationship source label, and relationship target constraints which do not constrain properties. label:default-output[]
+| LIST<STRING>
+
+| enforcedLabel
+| The implied labels required by a node label existence constraint or the source node and target node labels required by a relationship source or target label constraint, or `null` for other constraints which do not constrain labels. label:default-output[] label:cypher[Cypher 25 only] label:new[Introduced in Neo4j 2025.xx]
+| STRING
+
+| ownedIndex
+| The name of the index associated with the constraint or `null`, in case no index is associated with it. label:default-output[]
+| STRING
+
+| propertyType
+| The property type the property is restricted to for property type constraints, or `null` for the other constraints.
+label:default-output[]
+| STRING
+
+| classification
+| The status of a constraint in a graph type (`dependent`, `undesignated`, or `independent`). label:cypher[Cypher 25 only] label:new[Introduced in Neo4j 2025.xx]
+| STRING
+
+| options
+| The options passed to `CREATE` command, for the index associated to the constraint, or `null` if no index is associated with the constraint.
+| MAP
+
+| createStatement
+| Statement used to create the constraint, or `null` for constraints with `dependent` classification or if the constraint cannot be created through `CREATE CONSTRAINT` (for example in case of version incompatibilities).
```

Comment:
The reason for it being null is that it can't be created through create constraints, then both the version incompatibilities and dependent classifications are examples of why they can't be created.
```suggestion
| Statement used to create the constraint, or `null` if the constraint cannot be created through `CREATE CONSTRAINT` (for example in case of version incompatibilities or if for constraints with `dependent` classification).
```